### DISCUSSION TITLE
Ontological framework/developmental model

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#859] [Research]
Opening up this discussion thread as we are entering the 'data analysis' phase and considering how to categorize different orgs/practices/programs/language/'models', etc. @BoazBF @catherinet1 @rufuspollock What we have been saying a lot is something along the lines of 'developmental model', or how 'different organizations understand development', ie. what is the developmental model of this organization/how they understand development. Here I would suggest that we consider using the term 'ontological framework' as @rufuspollock pointed out in this thread: https://github.com/orgs/life-itself/discussions/839, and I think organizing our data analysis around this term might be a good idea, especially considering the following things:

- Some organizations (like the monastic academy) avoid the use of the term 'development.'
- Some organizations don't have a 'model' or specific way of understanding development, but instead a concept which organizes their practices program (like 'belonging') 
- It allows us to avoid thinking in linear ways (not all developmental models are linear, but it can have that connotation) 
- It can also help us illustrate what we were discussing on how there are different 'kinds of organizations' in the deliberately developmental spaces field, ones (like partners for youth) which aim on coaching people to enable deliberately developmental spaces, and ones that are more focused on the research part, like Kernels, and of course the straightforward dds that we are primarily concerned with

Some other things to consider though: 
- Does it make sense to consider 'SEL' and capacity building, as ontological frameworks? (I am inclined to say yes) maybe you have some clarifying thoughts on this @asimong

**Comment by asimong** [comment]
to me it would be clearer to say that SEL has or implies an ontological framework rather than is one. So one can ask the question, what is the ontological framework behind SEL? Is it, or is it not the same as for another approach?

**Reply by catherinet1** [reply]
i think that makes sense to me too, from a quick gut reaction

**Comment by rufuspollock** [comment]
Strictly it would be *ontogenetic* framework (i.e. these are frameworks about the evolution of being). However, that is kind of nitpicky and one could just use ontological.

**Reply by asimong** [reply]
Does this relate to the previously mentioned proposed distinction between descriptive and generative ontologies, @rufuspollock @matthewmccarthy11 ? Back on #839 that is? I think we're in line to do some useful meta-ontology here... Actually I rather like the term “ontogentic framework” as it makes more sense to me than “generative ontology”, if indeed the intention is the same.

**Reply by rufuspollock** [reply]
@asimong i don't think - i frankly didn't understand the generative vs descriptive ontologies.

to clarify:

- maps include both locations and how they relate to each other eg if one follows another and even how that unfolding or development occurs in a general sense (e.g. piaget's stages are a map)
- rafts describe activities that support growth (again in piagetian terminology we might say activities that generate  "accommodation")
- domains are a specific ontogenetic area

**Comment by rufuspollock** [comment]
> Some organizations (like the monastic academy) avoid the use of the term 'development.'

@matthewmccarthy11 could say more about this. what terms do they use?
